facilitator/src/sample.rs

```diff
@@ -133,7 +129,10 @@ pub fn generate_ingestion_sample(
                             .encode_simple(&data)
                             .context("failed to encode data")?;
 
-                        let r_pit = fake_server.choose_eval_at();
+                        // Hardcoded r_pit value
+                        // Some other options: 81133978, 998314904, 1103857271
+                        // Value obtained by running an instance of libprio::Server
```

Comment:
Rather than list other magic numbers, this comment should concisely explain why we're using a constant. "running an instance of libprio::Server" is kind of vague, too.